src/CMakeLists.txt

```diff
@@ -167,16 +166,16 @@ else()
 endif()
 
 target_woof_settings(woof)
-target_include_directories(woof PRIVATE
-        "${CMAKE_CURRENT_BINARY_DIR}/../" "../miniz")
+target_include_directories(woof PRIVATE "${CMAKE_CURRENT_BINARY_DIR}/../")
 
 target_link_libraries(woof PRIVATE
     ${SDL2_LIBRARIES}
     SDL2_net::SDL2_net
     OpenAL::OpenAL
     SndFile::sndfile
     opl
-    textscreen)
+    textscreen
+    miniz)
```

Comment:
What's the advantage of turning the two miniz source files into a static library?

Reply:
It just looks more natural in CMake. This is a leftover from the `git submodule` experiment. 
We also have static libraries `opl` and `texscreen`.